packages/web-components/docs/components/highlight.md

```diff
@@ -0,0 +1,44 @@
+# Data Commons Gauge Chart Web Component
+
+[Data Commons Web Component](../../README.md) for highlighting a specific
+variable value from a specific variable about a single place.
+
+## Usage
+
+```html
+<datacommons-highlight
+  title="Percentage of US Population that are Internet Users"
+  place="country/USA"
+  variable="Count_Person_IsInternetUser_PerCapita"
+  min="0"
+  max="100"
+></datacommons-highlight>
+```
+
+<img src="../assets/gauge.png" width="620"/>
```

Comment:
s/gauge/highlight/

Reply:
Thanks for the catch! Added a new image and linked to that instead.